docs_robots/solutions/husky_a300_observer/husky_a300_observer.mdx

```diff
@@ -100,6 +102,10 @@ import ComponentCommonSoftwareIntegration from "/components/common_software_inte
   <TOCInline toc={toc} />
 </details>
 
+## Quick Start
+
```

Comment:
Remove empty line, so `<ComponentAmpQuickstart />` is attached to `## Quick Start`